src/App.php

```diff
@@ -606,6 +606,22 @@ public function execute(Route $route, Request $request): self
                 }
             }
         } catch (\Throwable $e) {
+            if (empty(self::$errors['*'])) { // If no error handler is set then add a default one.
+                self::$errors['*'][] = [
+                    'callback' => function (\Throwable $error, Response $response) {
+                        $response->setStatusCode(500);
+                        $response->json([
+                            'message' => $error->getMessage(),
+                            'stacktrace' => $error->getTrace()
+                        ]);
+                        $response->send();
+                        \fwrite(STDERR, "\033[31mException: " . $error->getMessage() . "\033[0m\n");
+                        \fwrite(STDERR, "Stacktrace: \n" . $error->getTraceAsString() . "\n");
```

Comment:
Are there any side-effects of writing to a stream after using `response->send();`

Doesn't `response->send();` terminate the lifecycle of the request ?